packages/contracts/src/test/TestPaymasterConfigurableMisbehavior.sol

```diff
@@ -116,7 +116,7 @@ contract TestPaymasterConfigurableMisbehavior is TestPaymasterEverythingAccepted
         if (expensiveGasLimits) {
             uint sum;
             //memory access is 700gas, so we waste ~50000
-            for ( int i=0; i<60000; i+=700 ) {
+            for ( int i=0; i<100000; i+=700 ) {
```

Comment:
is this loop actually cheaper on hardhat? :-)

Reply:
Probably cheaper due to berlin and london gas costs update